pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -633,28 +633,28 @@ def test_lambda_named_agg(func):
 
 class TestLambdaMangling:
```

Comment:
move all of these tests to pandas/tests/test_aggregation.py

Reply:
just the tests where you are moving the code (e.g. the maybe_mangle.....) and such

Reply:
yeah, I was thinking it as a follow-up, but maybe good to have it in a go here! I moved all tests related to functions in aggregation to `pandas/tests/test_aggregation.py`